migrations/20200831174542_snapshot.sql

```diff
@@ -0,0 +1,10 @@
+CREATE TABLE snapshot (
+  filesystem_name TEXT NOT NULL,
+  snapshot_name TEXT NOT NULL,
+  create_time TIMESTAMP WITH TIME ZONE NOT NULL,
+  modify_time TIMESTAMP WITH TIME ZONE NOT NULL,
+  snapshot_fsname TEXT NOT NULL,
+  mounted BOOLEAN NULL,
+  comment TEXT NULL,
+  UNIQUE (filesystem_name, snapshot_name)
```

Comment:
nit: probably want to add an index for these combined fields as well.

Reply:
I think an index is created by definition:

```
chroma=> \d+ snapshot;
                                               Table "public.snapshot"
     Column      |           Type           | Collation | Nullable | Default | Storage  | Stats target | Description 
-----------------+--------------------------+-----------+----------+---------+----------+--------------+-------------
 filesystem_name | text                     |           | not null |         | extended |              | 
 snapshot_name   | text                     |           | not null |         | extended |              | 
 create_time     | timestamp with time zone |           | not null |         | plain    |              | 
 modify_time     | timestamp with time zone |           | not null |         | plain    |              | 
 snapshot_fsname | text                     |           | not null |         | extended |              | 
 mounted         | boolean                  |           |          |         | plain    |              | 
 comment         | text                     |           |          |         | extended |              | 
Indexes:
    "snapshot_filesystem_name_snapshot_name_key" UNIQUE CONSTRAINT, btree (filesystem_name, snapshot_name)
```

Reply:
Do you mean indexes for single columns?